frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -152,14 +152,34 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   }
   const os = iGetVmSettingValue(state, id, VMSettingsField.OPERATING_SYSTEM);
   const isWindows = os && os.startsWith('win');
+
+  dispatch(
```

Comment:
IMO, we should also consider the vm import part here.

can we change the value to false when isProviderImport? But keep it visible in case some import providers (possibly vmWare) wish to use it.

In case of ovirt provider it is actually impossible to use this field so can we hide it in `ovirt-state-update.ts`?

We should also register field reset for `MOUNT_WINDOWS_GUEST_TOOLS` in each provider - look for 
 vmFieldUpdate

Reply:
I don't have an env to test and check this scenarios ...

@suomiy can you take it as a follow up PR ?

Reply:
should be few lines, just use vmFieldUpdate in the same way like for other VM fields and also register isHidden in ovirt provider in the same spot/update.

I can test your PR, once you have it.

Reply:
added a gate to not mount when in import view